charts/datarepo-api/values.yaml

```diff
@@ -8,8 +8,9 @@ replicaCount: 1
 # It is set here, set in the api-deployment, and made an envvar that
 # overrides an application.properties item in DRmanager
 shutdownTimeoutSeconds: 60
-readinessInitialDelaySeconds: 90
-livenessInitialDelaySeconds: 90
+readinessInitialDelaySeconds: 10
+livenessInitialDelaySeconds: 10
+startupInitialDelaySeconds: 15
```

Comment:
It may be worthwhile talking to @tcjiang9 about the startup times, I believe some of this was modified for the Elasticsearch implementation. Otherwise, if there is no blocker lgtm!

Reply:
I think these were 90 for a long time (the last time the line changed was back in February)